manifest.toml

```diff
@@ -5,7 +5,7 @@ name = "Pixelfed Glitch"
 description.en = "Decentralized photo sharing social media platform - forked from Pixelfed"
 description.fr = "Plateforme de partage de photos décentralisée - dérivée de Pixelfed"
 
-version = "0.12.4+glitch.1.10.1~ynh1"
+version = "0.12.5~ynh1"
```

Comment:
Hmyeah the autoupdate is not gonna take care of having the `+glitch` prefix unfortunately